src/embed_tests/TestOperator.cs

```diff
@@ -0,0 +1,209 @@
+using NUnit.Framework;
+
+using Python.Runtime;
+
+using System.Linq;
+using System.Reflection;
+
+namespace Python.EmbeddingTest
+{
+    public class TestOperator
+    {
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            PythonEngine.Initialize();
+        }
+
+        [OneTimeTearDown]
+        public void Dispose()
+        {
+            PythonEngine.Shutdown();
+        }
+
+        public class SampleClass
+        {
+            public int VoidCall() => 10;
+
+            public int Foo(int a, int b = 10) => a + b;
+        }
+
+        public class OperableObject
+        {
+            public int Num { get; set; }
+
+            public OperableObject(int num)
+            {
+                Num = num;
+            }
+
+            public static OperableObject operator +(OperableObject a, OperableObject b)
+            {
+                return new OperableObject(a.Num + b.Num);
+            }
+            public static OperableObject operator +(OperableObject a, int b)
+            {
+                return new OperableObject(a.Num + b);
+            }
```

Comment:
Please, add an overload `OperableObject operator+ (int a, OperableObject b)` before these two to ensure it does not break adding two instances of `OperableObject`.

Reply:
ok, then I will add the equivalent for all the other operators as well. 

actually I found out that handling the reverse operators `+(int a, OperableObject b)` is just a matter of 1 line of code, should I just include that in this PR too along with the tests?

Reply:
Sure, if you think so.

Reply:
Actually nevermind the reverse operators was more involved, will leave for next PR